docs/modules/player-lifecycle/custom-player-creation.md

```diff
@@ -0,0 +1,127 @@
+* Directly modify `PlayerLifecycleConfig` to be false by default
+
+Once `AutoRequestPlayerCreation` is set to false you must manually call the `RequestPlayerCreation` method, which can be found in the `SendCreatePlayerRequestSystem`.
+
+```csharp
+var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+playerCreationSystem.RequestPlayerCreation();
+```
+
+To find out more about the `RequestPlayerCreation` method, take a look at the `SendCreatePlayerRequestSystem` [API reference documentation]({{urlRoot}}/api/player-lifecycle/send-create-player-request-system#methods).
+
+## Adding a player creation callback
+
+If manual player creation is enabled, you can configure a callback to be triggered when a player creation response is received. The callback must be of type `Action<PlayerCreator.CreatePlayer.ReceivedResponse>`.
+
+For example:
+
+```csharp
+void CallPlayerCreation()
+{
+    var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+    playerCreationSystem.RequestPlayerCreation(callback: OnCreatePlayerResponse);
+}
+
+void OnCreatePlayerResponse(PlayerCreator.CreatePlayer.ReceivedResponse response)
+{
+    if (response.StatusCode != StatusCode.Success)
+    {
+        Debug.LogWarning($"Error: {response.Message}");
+    }
+}
+```
+
+## Using arbitrary data in the player creation loop
+
+The default player creation loop sends requests automatically. To use arbitrary data in the loop, you must ensure that `AutoRequestPlayerCreation` is set to false. Some
+
+1. Disable `AutoRequestPlayerCreation`, some ways of doing this are described [here](#manually-sending-player-creation-requests).
+1. Call `RequestPlayerCreation` with serialized data.
+2. Deserialize data back into the original type.
+
+##### Call RequestPlayerCreation with serialized data
+
+Then, call `RequestPlayerCreation`. Ensure that your arbitrary data is serialized into a byte array.
+
+```csharp
+var myArguments = new SampleArgumentsObject
+{
+  PlayerName = "playerName",
+  SpawnPosition = new Coordinates(50, 0, 75))
+};
+var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+var serializedArguments = SerializeArguments(myArguments);
+playerCreationSystem.RequestPlayerCreation(serializedArguments);
+```
+
+Where an example implementation of `SerializeArguments` could be:
+
+```csharp
+private byte[] SerializeArguments(object playerCreationArguments)
+{
+    using (var memoryStream = new MemoryStream())
+    {
+        var binaryFormatter = new BinaryFormatter();
+        binaryFormatter.Serialize(memoryStream, playerCreationArguments);
+        return memoryStream.ToArray();
+    }
+}
+```
+
+##### Deserialize data into the same type you originally serialized from
+
+Lastly, ensure that you are deserializing the byte array into the same type of object you serialized it from. For example:
+
+```csharp
```

Comment:
There's a lot of code in here that isn't really relevant for a user.

The main bit of information that they care about is that the `playerCreationArguments` is the serialized byte array in the previous step and is given when your EntityTemplate delegate is called. That is never explicitly stated in the docs.

Reply:
Moved the serialize/deserialize sample code into expandables

I think that is good to continue the example from earlier with the `SampleArgumentsObject` so users have some example of how they could make use of the `byte[]` field